install_config/install/disconnected_install.adoc

```diff
@@ -0,0 +1,550 @@
+= Disconnected Installation
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+Frequently, portions of a datacenter may not have access to the Internet, even via proxy servers. Installing OpenShift in these environments is considered a disconnected installation.
+
+An OpenShift Enterprise disconnected installation differs from a regular
+installation two primary ways:
+
+* The OpenShift software channels and repositories are not available via Red Hat’s content distribution network. 
+* OpenShift uses several Dockerized components. Normally, these images are pulled directly from Red Hat’s Docker registry. In a disconnected environment, this is not possible.
+
+A disconnected installation ensures the OpenShift software is made available to the relevant servers, then follows the same installation process as a standard connected installation. This topic additionally details manually downloading the Docker images and how to transport them onto the relevant servers. 
+
+Once installed, in order to use OpenShift, you will need source code in a source control repository (for example, Git). This topic assumes that an internal Git repository is available that can host source code and this repository is accessible from the OpenShift nodes. Installing the source control repository is out of scope of this document.
+
+Also, when you try building applications in OpenShift, your build may have some external dependencies, such as a Maven Repository or Gem files for Ruby applications. By default, Red Hat docker images try to reach out to external repositories on the Internet. You can configure OpenShift to use your own internal repositories. For the purposes of this document, we assume that such internal repositories already exist and are accessible from the OpenShift nodes hosts. Installing such repositories is outside the scope of this document.
+ 
+[NOTE]
+====
+You can also have a http://www.redhat.com/en/technologies/linux-platforms/satellite[Satellite] server that provides access to Red Hat content via the intranet/LAN. For environments with Satellite, you can synchronize the OpenShift software into the Satellite for use with the OpenShift servers. 
```

Comment:
I don't understand "synchronize ... software into the Satellite" -- the "the" there seems out of place.